src/NHSD.BuyingCatalogue.Identity.Api/Models/ApplicationUser.cs

```diff
@@ -9,6 +9,8 @@ public sealed class ApplicationUser : IdentityUser
 
         public string LastName { get; set; }
 
+        public string DisplayName => $"{FirstName} {LastName}";
```

Comment:
I'd add a .Trim() to this, otherwise if there's no first name it'll be " {LastName}", and if there's no last name it'll be "{FirstName} "

Reply:
Although there is no database-level constraint that first name or last name cannot be empty (`NOT NULL` only) we do require a first name and last name in the application (`ApplicationUserValidator.ValidateFirstNameAsync` and `ApplicationUserValidator.ValidateLastNameAsync`) so a `.Trim()` shouldn't be necessary.

However, I don't mind adding one if we want to do 'belt and braces'.